de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/impl/HandlingUnitsBL.java

```diff
@@ -518,6 +519,25 @@ public boolean isTopLevel(final I_M_HU hu)
 		return topLevel;
 	}
 
+	@Override
+	public boolean isAnonymousHuPickedOnTheFly(@NonNull final I_M_HU hu, final ImmutableList<InOutLineId> lineIds)
```

Comment:
follow-up: not sure why we need `lineIds` as parameter?
if an HU is anonymousHuPickedOnTheFly, but for an unrelated inoutLine..how could the happen, and: you you then want to consider it *not* as anonymously picked?